Makefile

```diff
@@ -228,9 +250,12 @@ test-docker-mount-code:
 
 lint:              		  ## Run code linter to check code style and check if formatter would make changes
 	($(VENV_RUN); python -m ruff check --show-source . && python -m black --check .)
+	$(VENV_RUN); pre-commit run upgrade-deps-if-changed --files $(git diff master --name-only)
```

Comment:
The precommit doesn't really upgrade the requirements, right? And if so, I think it should be named the same way as in the make targets (where it's "reqs").